slugs.md

```diff
@@ -1945,7 +1945,6 @@ update the script at 'scripts/release/update-slugs-table.js'.
 | `Semaphore CI` | `semaphoreci` |
 | `SemVer` | `semver` |
 | `Sencha` | `sencha` |
-| `Sendinblue` | `sendinblue` |
```

Comment:
Don't need to edit the `slugs.md` file - this gets handled by the release :)

Reply:
You're right 😅 
I will bring this back!